src/daemon/plugin.c

```diff
@@ -153,13 +153,11 @@ static const char *plugin_get_dir(void) {
     return (plugindir);
 }
 
-static void plugin_update_internal_statistics(void) { /* {{{ */
-
+static int plugin_update_internal_statistics() { /* {{{ */
```

Comment:
Please keep the (void) in the argument list. While compilers still support K&R style, it has been deprecated since I think C89 ;)